### PR TITLE
Add Coverage Test badge for nntrainer

### PR DESCRIPTION
**Pull request by jijoongmoon**
Add Coverage Test Badge for nntrainer in README.md

Resolves:

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # NNtrainer
+[![Code Coverage](http://ec2-54-180-96-14.ap-northeast-2.compute.amazonaws.com/nntrainer/ci/badge/codecoverage.svg)](http://ec2-54-180-96-14.ap-northeast-2.compute.amazonaws.com/nntrainer/ci/gcov_html/index.html)
```
Comment (dongju-chae) [inline README.md:2]: BTW, it seems that `http://nnsuite.mooo.com/nntrainer` is not working. Is it different from `http://nnsuite.mooo.com/nnstreamer`?

Comment (wooksong) [reply README.md:2]: @dongju-chae NNT uses the aws-dev server not the aws server, nnsuite.mooo.com.

Comment (dongju-chae) [reply README.md:2]: Got it

Comment (dongju-chae) [reply README.md:2]: BTW, it would be great if we have an additional URL alias such as `nnsuitedev.mooo.com`

**Pull request merged**